README.md

```diff
@@ -128,6 +128,36 @@ There is also a [tapioca-wrapper](https://github.com/vintasoftware/tapioca-wrapp
 
 ### Using Docker
 
```

Comment:
Would you mind rolling back the changes on `README.md`? (see [my previous comment](https://github.com/datasciencebr/jarbas/pull/85#issuecomment-268895808))